scripts/effects/effect-helpers.js

```diff
@@ -140,4 +140,64 @@ export default class EffectHelpers {
 
     return activeEffect.update({ statuses: statusesArray });
   }
+
+  /**
+   * Migrates the description of Custom Convenient Effects to use the ActiveEffect5e#description field instead of the legacy flags.
+   *
+   * @param {string} id - the ID of the custom effects item
+   * @returns {Promise} a promise that resolves when all the Item's ActiveEffects descriptions have migrated
+   */
+  async _migrateCustomEffectsItemEffectDescriptions(id, migration) {
+    const item = this._settings._findCustomEffectsItem(id);
+    const updates = [];
+    const scope = Constants.MODULE_ID;
```

Comment:
Assigning a variable when you could just use the constant as is seems redundant. Remove all `const x = Constants.X` in favor of just using `Constant.X`